server/config.js

```diff
@@ -13,7 +13,7 @@ const convictConfig = convict({
     password: {
       doc: 'DB Password',
       env: 'DB_PASSWORD',
-      default: 'postgres',
+      default: 'postres',
```

Comment:
Don't change the default. You can add this as an enviroment variable for the PC, or you can create `.env` file

Reply:
Oops, my bad